change/@ni-nimble-components-b40541e2-0d52-423d-a278-53d68df3aa8e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There is an unrelated chromatic diff present in the nimble-select (only in Chrome) that I'm curious if anyone has any ideas about. I'm not sure how this code change could have caused that diff or why the diff is only present in the Dark Theme in Chrome, but the diff has persisted across two runs of the chromatic tests.